WINDSURF_SETUP.md

```
@@ -0,0 +1,247 @@
+```
+
+### 3. Test the Stdio Server (Optional)
+
+You can test that the server starts correctly:
+
+```bash
+npm run stdio
+```
+
+You should see:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)